pkg/types/attributes.go

```diff
@@ -91,6 +91,10 @@ const (
 	AttrIconFlip = "flip"
 	// AttrUnicode local libasciidoc attribute to encode output as UTF-i instead of ASCII.
 	AttrUnicode = "unicode"
+	// AttrOptions element options (boolean, comma separated)
+	AttrOptions = "options"
+	// AttrOpts alias for AttrOptions
+	AttrOpts = "opts"
```

Comment:
why do we need such an alias?

Reply:
It's documented in asciidoctor.

Reply:
oh ok, I missed that.

Reply:
I've got a change (not here, will be follow up) that enables this grammar for all the blocks.  that use Attributes.  It doesn't yet add the support for this with [verse] and similar *yet*, but that's coming soon.  The main thing holding me back is writing all the *new* test cases.

(I also had to fix some tests, since adding the new syntax rules broke some of the tests that were actually incorrect and reflected implementation rather than documented syntax.)